apps/telegram-bot/core/services/outbox_consumer.py

```
@@ -0,0 +1,99 @@
+
+
+async def _send_error_to_chat(chat_id: int | str, error: str) -> None:
+    """Send an error notification to the user's chat."""
+    try:
+        from core.services.bot_app import application
+
+        await application.bot.send_message(
+            chat_id=chat_id,
+            text=f"Sorry, an error occurred while processing your request:\n\n{error}",
+        )
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)